src/api/type-defs/objects/MLModel.ts

```diff
@@ -10,6 +10,6 @@ export default /* GraphQL */ `
     description: String
     version: String!
     defaultConfThreshold: Float
-    categories: [Categories]
+    categories: [Categories!]
```

Comment:
@alukach I think the plural thing was a typo:

```
  type Categories { // this should be Catagory
    _id: String!
    name: String!
    color: String!
  }
```

and the `type MLModel` should be (`categories` should also be required):
```
categories: [Category!]!
```

Reply:
Yes please! Thanks @alukach

Reply:
@nathanielrindlaub Okay, done ✅

Reply:
Mind adding one more explaination mark to make `categories` required? I know that wasn't the case before but MLModels should always have categories (see note above). 

```
categories: [Category!]!
```

Reply:
Sounds good @nathanielrindlaub , see https://github.com/tnc-ca-geo/animl-api/pull/198/commits/3ff48e228ad0cd63fa28fa756d4295b1bf5a5e0b